src/error.ts

```diff
@@ -0,0 +1,47 @@
+interface FirebaseErrorOptions {
+  children?: unknown[];
+  context?: unknown;
+  exit?: number;
+  original?: Error;
+  status?: number;
+}
+
+const DEFAULT_CHILDREN: NonNullable<FirebaseErrorOptions["children"]> = [];
+const DEFAULT_EXIT: NonNullable<FirebaseErrorOptions["exit"]> = 1;
+const DEFAULT_STATUS: NonNullable<FirebaseErrorOptions["status"]> = 500;
+
+export class FirebaseError extends Error implements FirebaseErrorOptions {
```

Comment:
what happened to the `stack` property?

Reply:
It exists on native `Error`, which this class extends.